pkgs/tools/misc/fortune/update.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl common-updater-scripts jq
+
+version="$(curl "https://api.github.com/repos/shlomif/fortune-mod/releases" | jq '.[0].name' --raw-output)"
```

Comment:
Since the releases are apparently on GitHub, wouldn't it be better to use `fromFromGitHub` for the `src` attribute?

Reply:
By changing `src` to `fetchFromGitHub`, the update script should also not be necessary anymore, since nixpgs-update/r-ryantm check GitHub releases as well.

Reply:
The github repo doesn't contain something in the release one, which are mainly about document. We can build them from the source but it will invole many other building tools which are not on nixpkgs(some perl packages and others). Furthermore, the build process is a bit complicated.

Reply:
> The github repo doesn't contain something in the release one,

Ah, that's unfortunate. But I think the nixpkgs-update logic also works when getting archives from GitHub releases with `fetchurl`. So I think it's best to fetchurl this from GitHub and then put a small comment above `src` why `fetchurl` is used rather than `fetchFromGitHub`.